Russel -> Russell (#114)
InnerSourceCommons/innersourcecommons.org: events/isc-spring-2020.md

Welcome to the 10th InnerSource Commons taking place in Madrid, Spain!

### COVID-19 and InnerSource Commons current Status

> The InnerSource Commons Madrid Summit is taking place at this moment of time. We suggest you follow general recommendations by the WHO and once in Spain the Spanish government ones. We are updating the [COVID-19 related page](https://innersourcecommons.org/events/isc-spring-2020-coronavirus/) with the last news about the situation in Madrid.

Reviewer: ```suggestion
As of now, the InnerSource Commons Madrid Summit is taking place. We suggest you follow general recommendations by the WHO and - once in Spain - those of the Spanish government. We are constantly updating the [COVID-19 related page](https://innersourcecommons.org/events/isc-spring-2020-coronavirus/).
```

All of them are tidy, clean, have wifi, air conditioning, and each room has its own bathroom (but double check).

> ### Call For Sponsorship

Reviewer: Is it an accidental commit or should this be added with this PR as well? (It's allright if you want to add it directly, just asking to double check)

Reviewer 2: Well, in reality this should have taken place in another commit... but then I modified the main site and move some paragraphs down to add the COVID-19 statement on the top.

Reviewer: 👍
